docs/tutorial-modules/varjo.mdx

```diff
@@ -0,0 +1,76 @@
+:::
+
+### Installation
+
+To use the VRCFTVarjoModule, you will need a compatible headset and the Varjo Base installed.
+You can get a copy of the Varjo Base from Varjo's website, you will need an account to download it. 
+
+### Hardware
+
+The VRCFTVarjoModule is compatible with the following hardware:
+
+| Headset | Eye Tracking | Facial Tracking |
+| ---------------------------------- | -------- | --------- |
+| Varjo Aero                         | `✅` Yes | `❌` No  |
+| Varjo XR-3*                        | `✅` Yes | `❌` No  |
+| Varjo VR-3*						 | `✅` Yes | `❌` No  |
+
+*Untested
+
+## Using The Varjo Module
+
+To use your Varjo with VRChat, you will need to use one of two modules with the VRCFaceTracking software:
+
+- [M3ga's](https://github.com/m3gagluk/VRCFTVarjoModule) is considered to be the more "stable" option, but only has binary blinking (eyes are fully open and closed)
+- [Chickenbread's](https://github.com/Chickenbreadlp/VRCFTVarjoModule) is a experimental fork of M3ga's module. It supports variable eye openness available in newer Varjo SDK versions
+
+For more information on modules and how to use them, see [Modules](./modules.mdx).
+
+:::warning
+You MUST unblock the VRCFTVarjoModule.dll file after downloading it. This can be done by right-clicking the VRCFTVarjoModule.dll file, selecting "Properties", and then selecting the "Unblock" button.
+If you do not see this option, you do not need to worry about this step.
+:::
+<div style={{
+    width: '50%',
+    height: 'auto',
+    margin: 'auto',
+    display: 'block'
+}}>
+    <img src={require("./img/varjo/varjo_unblock_instructions.png").default} alt="varjo unblock instructions" />
+</div>
+
+Assuming there are no connection issues with the hardware, the connection hardware components should be initialized by VRCFT after a few seconds.
+
+### Calibrating Eye Tracking
+
+Eye tracking is calibrated automatically when you put on the headset, as a quick 1-point calibration.
+Additionally, you can perform a 5-point calibration from the leftmost tab in the Varjo Base.
+<div style={{
+    width: '50%',
+    height: 'auto',
+    margin: 'auto',
+    display: 'block'
+}}>
+    <img src={require("./img/varjo/varjo_unblock_instructions.png").default} alt="varjo unblock instructions" />
+</div>
+Eye tracking calibration is not required to be performed frequently, but should be done after the initial Varjo Base software installation, after a change to the headset (i.e. change the facial interface), or when switching the headset between users.
+
+Start calibration by making sure "Allow eye tracking" is enabled in the System tab, then hit the "Calibrate eye tracking" button in the app. Follow the in-app instructions to finish calibrating the eye gaze tracking.  
+
+## Uninstalling The VRCFTVarjoModule
+
+To uninstall The VRCFTVarjoModule, navigate to `%\AppData\Roaming\VRCFaceTracking\CustomLibs` and delete the file called VRCFTVarjoModule.dll.
```

Comment:
This path is invalid. You could put something like `%appdata%\VRCFaceTracking\CustomLibs` instead

Reply:
Resolved in #8c4eb085.

Reply:
This is not resolved as of latest (https://github.com/VRCFaceTracking/docs/pull/3/commits/8c4eb085a4e08d000ca95291bb4f755e2790d378)